packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -71,6 +71,11 @@ type SummaryGeneratorOptionalTelemetryProperties =
     "opsSinceLastAttempt" |
     /** Delta between the current reference sequence number and the reference sequence number of the last summary */
     "opsSinceLastSummary" |
+    /** Delta in sum of op sizes between the current reference sequence number and the reference
```

Comment:
I'd love to see proper interface with types here. I'm not sure there is any benefit to current structure. @pleath ?